app/features/flex.test.js

```diff
@@ -92,6 +92,81 @@ test('Can apply space-between', async t => {
   t.pass()
 })
 
+test('Can adjust wrapping', async t => {
+  const { page } = t.context
+
+  await page.click(test_selector)
+  await page.keyboard.down('Control')
+  await page.keyboard.down('Shift')
+  await page.keyboard.press('ArrowUp')
+  let wrapStr = await page.$eval(test_selector, el => el.style.flexWrap)
+  t.true(wrapStr === 'nowrap')
+
+  await page.keyboard.press('ArrowUp')
+  wrapStr = await page.$eval(test_selector, el => el.style.flexWrap)
+  t.true(wrapStr === 'nowrap')
+
+  await page.keyboard.press('ArrowDown')
+  wrapStr = await page.$eval(test_selector, el => el.style.flexWrap)
+  t.true(wrapStr === 'wrap')
+
+  await page.keyboard.press('ArrowDown')
```

Comment:
is this to test the clamping logic?

Reply:
yes, it's supposed to test if correct value was applied and also clamping so when correct value is already applied it doesn't apply something else. It may have more sense to make separated tests, I wasn't sure.